### DISCUSSION TITLE
I get 401 from the request

### DISCUSSION THREAD

**Discussion by rarytic** [vpnhood/VpnHood#576] [Q&A] [answered]
I do https://api.vpnhood.com/api/v1/projects/Pid/access-tokens/Aid/access-key
but I always get 401, what am I doing wrong?

**Comment by trudyhood** [comment]
Have you created your API key?
Go to Team, Add a "Bot," and get your API key. You need to pass it as a standard Bearer authorization in your request.

**Comment by rarytic** [comment]
Thank you very much for your answer!
Now I'm doing https://api.vpnhood.com/api/v1/authentication/signin.
And I convey
{
  "idToken": "Bearer token",
  "refreshTokenType": "None"
}
I get 401 and
{
  "TypeName": "AuthenticationException",
  "TypeFullName": "System.Security.Authentication.AuthenticationException",
  "Message": "IDX12709: CanReadToken() returned false. JWT is not well formed.\nThe token needs to be in JWS or JWE Compact Serialization Format. (JWS): 'EncodedHeader.EncodedPayload.EncodedSignature'. (JWE): ' EncodedProtectedHeader.EncodedEncryptedKey.EncodedInitializationVector.EncodedCiphertext.EncodedAuthenticationTag'.",
  "Data": {}
}
without "Bearer " i get "This is not an id token".

**Comment by trudyhood** [comment, marked as answer]
You don't need authorization or authentication, like sign-in to call API. Using VpnHood API is simple and easy. Just follow this instruction
1. https://console.vpnhood.com/ via browser, 
2. Go to Team and create a "Bot"
3. Copy its API key
4. Use the API key as a standard bearer authorization to call any API
